db/schema.rb

```diff
@@ -45,10 +50,32 @@
     t.index ["scheduled_at"], name: "index_good_jobs_on_scheduled_at", where: "(finished_at IS NULL)"
   end
 
-  create_table "schools", force: :cascade do |t|
+  create_table "institution_types", force: :cascade do |t|
     t.string "name"
+    t.integer "rspo_institution_type_id"
```

Comment:
Those probably cannot be null

Reply:
yes and there is a foreign key placed on this field that guarantees null safety

Reply:
I see there're not null constraints in schema now, but just FYI, foreign key constraints ensure reference consistency, but they're separate from not null constraints. If you wanted to have an optional association, you'd add a foreign key constraint, but you wouldn't add a not-null constraint.